Sources/MediaViewControllers/PlaylistViewController.swift

```diff
@@ -0,0 +1,33 @@
+/*****************************************************************************
+ * VLCPlaylistViewController.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Carola Nitz <caro # videolan.org>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+class VLCPlaylistViewController: VLCMediaViewController {
+    override init(services: Services) {
+        super.init(services: services)
+        setupUI()
+    }
+
+    private func setupUI() {
+        title = NSLocalizedString("PLAYLISTS", comment: "")
+        tabBarItem = UITabBarItem(
+            title: NSLocalizedString("PLAYLISTS", comment: ""),
+            image: UIImage(named: "TVShowsIcon"),
+            selectedImage: UIImage(named: "TVShowsIcon"))
+        tabBarItem.accessibilityIdentifier = VLCAccessibilityIdentifier.playlist
+    }
+
+    override func viewControllers(for pagerTabStripController: PagerTabStripViewController) -> [UIViewController] {
+        return [
+            VLCPlaylistCategoryViewController(services)
```

Comment:
Is there a way to disable `PagerTabStrip...` when there is only one VC, I think it will look kind of odd with one item?

Reply:
I actually wouldn't want to do that right now and discuss it with Louis. Right now it looks alike and in sync with the other controllers

Reply:
I'm sorry if I misunderstood but, are you saying we should merge and see later on with Louis or we should freeze this PR until Louis input?

I agree that his input is important but on some cases I think we can be independent.
I believe this case is one of them mainly due to:

 - Redundancy of Playlists "label" on top
 - Scroll on sides
 - Unnecessary usage of top space for a single item

![gifplaylist](https://user-images.githubusercontent.com/14984800/46339947-63d06180-c634-11e8-82a7-4ca7f0b62232.gif)

I'm always open for discussion if you have other opinions on the matter.

Reply:
The bouncing should definitely be disabled but I wouldn't want to remove the top above. I'd rather just change the text.  I can see already issues in regards to editbar positioning or emptyview positioning which is done the same on video, audio and playlistviewcontrollers and the little bit of space that we loose is okay if the behavior and UI is consistent
And yes besides the bouncing I would address the rest in a follow up commit